Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -102,6 +102,14 @@ def IsCyclic.commGroup [hg : Group α] [IsCyclic α] : CommGroup α :=
 
 variable [Group α]
 
+/-- A non-cyclic multiplicative group is non-trivial.
+The additive version is `not_isCyclic.Nontrivial'`. -/
+@[to_additive not_isCyclic.Nontrivial' "A non-cyclic additive group is non-trivial.
+The multiplicative version is `not_isCyclic.Nontrivial`."]
+theorem not_isCyclic.Nontrivial (nc : ¬IsCyclic α) : Nontrivial α := by
```

Comment:
It looks like this should be called
```suggestion
theorem not_isCyclic.nontrivial (nc : ¬IsCyclic α) : Nontrivial α := by
```
or
```suggestion
theorem Nontrivial.of_not_isCyclic (nc : ¬IsCyclic α) : Nontrivial α := by
```
because `not_*.nontrivial` doesn't work with dot notation. Also, the additive version should use `not_isAddCyclic`, not `nontrivial'`.

Reply:
Thanks Yuri, I went for the `Nontrivial.of_not_isCyclic`.

I wonder whether `isCyclic --> isAddCyclic` is something that I should teach to `to_additive`.